schema/examples/direct/multiple_signers/status-response.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<direct-signature-job-status-response xmlns="http://signering.posten.no/schema/v1">
+    <signature-job-id>1</signature-job-id>
+    <status signer="12345678910">SIGNED</status>
+    <status signer="10987654321">SIGNED</status>
+    <confirmation-url>https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/complete</confirmation-url>
+    <xades-url signer="12345678910">https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/xades/1</xades-url>
+    <xades-url signer="10987654321">https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/xades/2</xades-url>
+    <pades-url>https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/pades</pades-url>
+</direct-signature-job-status-response>
```

Comment:
Ja – enig i at det hadde vært nyttig med en aggregert status for hele oppdraget. For å beholde bakoverkompatibiliteten må vi vel legge til et nytt felt for det (f.eks. `job-status`) og så fortsette med at `status` (med optional `signer`-attributt) har informasjon om hver enkelt signatars status.

Også enig i at det bør være en streng – det er vel en uproblematisk endring gitt at streng-verdiene er de samme vi har i enum-en i dag.

Slenger sammen et forslag.